addons/mail/static/src/core/common/thread.xml

```diff
@@ -92,9 +90,8 @@
 </t>
 
 <t t-name="mail.NotificationMessage">
-    <div class="o-mail-NotificationMessage text-break mx-auto text-500 small px-3 text-center" t-on-click="onClickNotification" t-att-class="{
-        'mt-1': prevMsg and !prevMsg.isNotification,
-    }" t-ref="root">
+    <!-- @this {common/message} -->
+    <div class="o-mail-NotificationMessage text-break mx-auto text-500 small px-3 text-center" t-on-click="onClickNotification" t-att-class="props.className" t-ref="root">
```

Comment:
It's easier to see that as doc of template `mail.NotificationMessage`, so move it above.

Also instead of simply `{common/message}`, I would copy how we do in JS:
```xml
<!-- @this {import("@mail/core/common/message"} -->
```

It's both very clear and we are used to read and understand it in rest of code, so the current weirdness of template staying there is a bit less intimidating.